GoogleMapsApi/Entities/Geocoding/Request/GeocodingComponents.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+
+namespace GoogleMapsApi.Entities.Geocoding.Request
+{
+    public class GeocodingComponents : Dictionary<string, string>
```

Comment:
I had doubt in this case. I used this inheritance just to let flexible, case Google implements a new filter component, we can only use Dictionary.Add. But i can adjust it.